src/data/sentinel.json

```diff
@@ -12,7 +12,7 @@
   "metadata": {
     "title": "Sentinel by HashiCorp",
     "description": "Sentinel is a language and framework for policy built to be embedded in existing software to enable fine-grained, logic-based policy decisions.",
-    "image": "/sentinel/img/og-image.png",
+    "image": "https://docs.hashicorp.com/sentinel/img/og-image.png",
```

Comment:
Note: I'm not sure the open graph image for Sentinel will work yet, as I can't seem to access anything in Sentinel's public folder (favicon.svg is the only thing I can test, though) 🤔

Do we need some extra config to expose folders? Eg I can access https://www.consul.io/consul/_favicon.ico, but not https://docs.hashicorp.com/sentinel/favicon.svg . Seems potentially related to `vault-public` / `waypoint-public` naming? cc @BRKalow

Reply:
@zchsh I think there might be something funky going on with sentinel as we have a redirect in place to send all docs.hashicorp.com traffic to docs.hashicorp.com/sentinel 💭

Reply:
Ah that makes sense... do you think it makes sense to merge this PR as-is, and follow up on Sentinel?

Reply:
Yeah, that sounds great to me 😃